packages/mdx/lib/plugin/recma-document.js

```diff
@@ -102,10 +102,10 @@ export function recmaDocument(options) {
     /* c8 ignore next -- comments can be missing in the types, we always have it. */
     if (!tree.comments) tree.comments = []
 
-    if (pragmas.length > 0) {
+    for (let index = pragmas.length; index--; index >= 0) {
       tree.comments.unshift({
         type: 'Block',
-        value: pragmas.join(' '),
+        value: pragmas[index],
         data: {_mdxIsPragmaComment: true}
       })
     }
```

Comment:
Nitty, but I don‘t really like the reverse stuff.
This works locally (but I can’t suggest it on GH as it overlaps):

```js
    tree.comments.unshift(
      ...pragmas.map(function (d) {
        return /** @type {const} */ ({
          type: 'Block',
          value: d,
          data: {_mdxIsPragmaComment: true}
        })
      })
    )
```

Only don’t really like the `const` but TS needs it apparently

Reply:
Ahh. I dunno maybe a lot of repetition. Maybe more comments will be added in the future too.
I don‘t mind my const too much? How do you feel about that?

Reply:
I already applied this using a function to avoid repetition. I like it, but I can revert if you don’t.